src/frontend/Komponenter/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/KontantstøtteAlert.tsx

```diff
@@ -9,18 +12,70 @@ const AlertStripe = styled(Alert)`
 
 interface Props {
     harKontantstøttePerioder?: boolean;
+    kontantstøttePerioderGrunnlagsdata?: KsPeriode[];
 }
-export const KontantstøtteAlert: React.FC<Props> = ({ harKontantstøttePerioder }) => {
+
+const kontantstøtteKilde = (kilde: string): string => {
+    return kilde.toLowerCase();
+};
+
+export const KontantstøtteAlert: React.FC<Props> = ({
+    harKontantstøttePerioder,
+    kontantstøttePerioderGrunnlagsdata,
+}) => {
     if (harKontantstøttePerioder === null || harKontantstøttePerioder === undefined) {
         return null;
     }
 
     return (
         <>
-            <AlertStripe variant={harKontantstøttePerioder ? 'warning' : 'info'} size={'small'}>
+            <AlertStripe variant="info">
                 {harKontantstøttePerioder
-                    ? 'Bruker har eller har fått kontantstøtte.'
-                    : 'Bruker har verken fått eller får kontantstøtte.'}
+                    ? 'Brukers kontantstøtteperioder'
```

Comment:
Hvordan blir dette når `harKontantstøttePerioder` er true, men `kontantstøttePerioderGrunnlagsdata` er null? Da blir kanskje denne teksten litt pussig og bør være noe ala den gamle?

Reply:
Jeg lurte på om man har behov for både `harKontantstøttePerioder` og `kontantstøttePerioderGrunnlagsdata`? Kan man fase ut `harKontantstøttePerioder` og sjekke på periodene, eller blir dette feil?

Reply:
Vi kan godt fase ut harKontantstøttePerioder tenker jeg